anago

```diff
@@ -1488,13 +1488,7 @@ push_all_artifacts () {
 
 BASEDIR=${FLAGS_basedir}
 if [[ -z "${BASEDIR}" ]]; then
-  # Goobuntu machines have a standard path for "local" disk as the home
-  # directory is often stored elsewhere (NFS or something more exotic.)
-  if [[ -d "/usr/local/google" ]]; then
-    BASEDIR="/usr/local/google/$USER"
```

Comment:
This was redundant on my workstation, FWIW.
It can be overridden by the flag still anyhow so 🤷‍♂️

Reply:
Yay!

Reply:
... actually scratch that it's `/usr/local/google/home/bentheelder`, but again, we have the override.